src/components/Menu/MenuItem.tsx

```diff
@@ -162,6 +166,16 @@ class MenuItem extends UIComponent<Extendable<IMenuItemProps>, IMenuItemState> {
                 render: renderIcon,
               })}
             {content}
+            {endIcon &&
+              Icon.create(this.props.endIcon, {
+                defaultProps: {
+                  xSpacing: !!content ? 'before' : 'none',
+                  style: {
+                    float: 'right',
```

Comment:
[RFC]: 
I am aware that this is not the best way to make the icon render at the end of the menuItem. Can someone can help me with how to achieve this thing correctly? CSS noob here.